Library/Homebrew/test/test_integration_cmds.rb

```diff
@@ -193,7 +193,7 @@ def test_cellar_formula
   end
 
   def test_env
-    assert_match %r{CMAKE_PREFIX_PATH="#{HOMEBREW_PREFIX}[:"]},
+    assert_match /CMAKE_PREFIX_PATH="#{HOMEBREW_PREFIX}[:"]/,
```

Comment:
Nothing wrong here, but now that you're touching this it would good to escape the embedded string, i.e.:

```ruby
/CMAKE_PREFIX_PATH="#{Regexp.escape(HOMEBREW_PREFIX)}[:"]/
```

(This is generally a good idea to avoid misinterpretation of characters with special meaning in regular expressions. And we don't know in advance whether `HOMEBREW_PREFIX` might contain such problematic characters.)